liquibase-core/src/main/java/liquibase/example/StartH2Main.java

```diff
@@ -48,7 +49,13 @@ public static void main(String[] args) throws Exception {
             long seconds = 60;
             long millis = 1000;
             while (true) {
-                Thread.sleep(seconds * millis);
+                try {
+                    Thread.sleep(seconds * millis);
+                } catch (InterruptedException interruptedException) {
+                        System.err.println("Error starting H2");
```

Comment:
@riakivets Can you remove System.errr?

Reply:
@suryaaki2 I can remove it. What is your concern to keep it?